templates/taxi/driver_list.html

```diff
@@ -6,7 +6,15 @@ <h1>Driver list</h1>
     <ul>
       {% for driver in driver_list%}
         <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+          <a href="{% url "taxi:driver-detail" pk=driver.id %}">
```

Comment:
There is an extra space at the end of the `href` attribute in the URL tag. It should be `href="{% url 'taxi:driver-detail' pk=driver.id %}"` without the space to ensure the URL is correctly formed.